build/base/mpich-builder.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM debian:bullseye as builder
```

Comment:
Ideally, we should be using a newer version, but given that `mpioperator/base` still uses bullseye, this makes sense.

If you have the chance, I would welcome a PR to update the base image everywhere.

Reply:
Noted. Happy to do it in a separate PR.